.github/workflows/build.yml

```diff
@@ -23,12 +25,6 @@ jobs:
           POSTGRES_DB: "tempdb"
         ports:
           - 5432:5432
-        # Set health checks to wait until postgres has started
-        options: >-
-          --health-cmd pg_isready
```

Comment:
PG is ready in < 5 seconds, we don't use PG for another 30 seconds. safe to not check here & save a few seconds